typescript-express-reviews/src/api/Vehicle/findById.ts

```diff
@@ -3,7 +3,7 @@ import Vehicle from '../../models/vehicle';
 import Review from '../../models/review';
```

Comment:
Why is this file `Vehicle/findById` instead of `Review/findByVehicle`?

Reply:
https://trello.com/c/ShGHbcB0/87-mongoosejs-sample-apps-vehicle-review-app-with-typescript-express
#5 on the list